src/Drupal/Migrate/ValidateMigrationId.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Drush\Drupal\Migrate;
+
+use Attribute;
+use Consolidation\AnnotatedCommand\Parser\CommandInfo;
+
+#[Attribute(Attribute::TARGET_METHOD)]
+class ValidateMigrationId
```

Comment:
You can now extend NoArgumentBase to save a little code. See src/Attributes/OptionsetProcBuild.php for an example